### PR TITLE
Add Header and Side Navbar along with setting up Tailwind

### LINKED ISSUE #14: Add header and side navigation bar for traversing to different sections.

### PR DESCRIPTION
**Pull request by dhairyaj**
Closes #14 

Implementation
1. Added Tailwind CSS
2. Added react-icons package
3. Modified folder structure and file names
4. Added Header and sidebar components
5. Added some basic styles and a placeholder for Logo and theme toggle functionality

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
-import "./App.css";
-
-import InputSide from "./components/InputSide/InputSide";
-import OutputSide from "./components/OutputSide/OutputSide";
+import LeftPanel from "./components/LeftPanel/LeftPanel";
+import RightPanel from "./components/RightPanel/RightPanel";
 
 function App() {
```
Comment (frankhart2018) [inline frontend/src/App.js]: Let's use arrow functions to maintain consistency.

File: frontend/src/components/LeftPanel/Header.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BsFillSunFill, BsFillMoonFill } from "react-icons/bs";
+
+function Header() {
```
Comment (frankhart2018) [inline frontend/src/components/LeftPanel/Header.js]: Arrow function

File: frontend/src/components/LeftPanel/LeftPanel.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import NavBar from './NavBar';
+import Header from './Header';
+
+function LeftPanel() {
```
Comment (frankhart2018) [inline frontend/src/components/LeftPanel/LeftPanel.js]: Arrow function

File: frontend/src/components/LeftPanel/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { FaUserAlt } from "react-icons/fa";
+function NavBar() {
```
Comment (frankhart2018) [inline frontend/src/components/LeftPanel/NavBar.js]: Arrow function

File: frontend/src/components/LeftPanel/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      {/* TODO: Need to prepare a logo and favicon */}
+      <div className='w-full h-16 border-2 border-orange-300'>
+        Logo
+      </div>
+      <div className='h-4/6 my-auto flex flex-col justify-evenly items-center'>
+
+        {/* TODO:  Need to provide icons for following
+        Basic details, address, social media, work ex, education, awards, publications, certifications, skills, languages, interests, projects, references */}
+
+        <FaUserAlt />
```
Comment (frankhart2018) [inline frontend/src/components/LeftPanel/NavBar.js]: Will there be links that move the view to that location as well? If so, then let's just put all the icons and links in an array of objects and map that here, hard-coded values make it inflexible to add more sections in the future.

The object can look like:

```js
{
    "href": "some-link",
    "icon": "icon component",
}
```

File: frontend/src/components/RightPanel/RightPanel.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function RightPanel() {
```
Comment (frankhart2018) [inline frontend/src/components/RightPanel/RightPanel.js]: Arrow function

File: frontend/src/App.js (hunk 1 above)
Comment (dhairyaj) [reply frontend/src/App.js]: Done

File: frontend/src/components/LeftPanel/Header.js (hunk 2 above)
Comment (dhairyaj) [reply frontend/src/components/LeftPanel/Header.js]: Done

File: frontend/src/components/LeftPanel/LeftPanel.js (hunk 3 above)
Comment (dhairyaj) [reply frontend/src/components/LeftPanel/LeftPanel.js]: Done

File: frontend/src/components/LeftPanel/NavBar.js (hunk 4 above)
Comment (dhairyaj) [reply frontend/src/components/LeftPanel/NavBar.js]: Done

File: frontend/src/components/LeftPanel/NavBar.js (hunk 5 above)
Comment (dhairyaj) [reply frontend/src/components/LeftPanel/NavBar.js]: Seems like a good idea.

File: frontend/src/components/RightPanel/RightPanel.js (hunk 6 above)
Comment (dhairyaj) [reply frontend/src/components/RightPanel/RightPanel.js]: done

File: frontend/src/components/LeftPanel/NavBar.js (hunk 5 above)
Comment (dhairyaj) [reply frontend/src/components/LeftPanel/NavBar.js]: Done

**Comment by dhairyaj** [comment]
Implementation changes in 2 updated commits
   - Formatted the code using prettier to maintain the code formatting standard
   - Modified the components using arrow functions
   - Added Material UI dependencies ( we will use material ui components and if needed tailwind)
   - Added a array of objects for the navigation menu items

**Review by frankhart2018 — APPROVED** [review]
LGTM

**Pull request merged**